tests/tests/cpld_debug_noaero_p8

```diff
@@ -0,0 +1,92 @@
+#
+#  cpld_debug_p8 test
```

Comment:
maybe change it to cpld_debug_p8 no aero test to keep consistent with the test name.